Description: Use Debian help path
 This patch fixes Dia to use Debian's help directory for documentation
Author: Roland Stigge <stigge@antcom.de>

--- dia-0.97.1.orig/app/commands.c
+++ dia-0.97.1/app/commands.c
@@ -632,16 +632,14 @@ edit_redo_callback (GtkAction *action)
 void
 help_manual_callback (GtkAction *action)
 {
-#ifdef GNOME
-  gnome_help_display("dia", NULL, NULL);
-#else
   char *helpdir, *helpindex = NULL, *command;
   guint bestscore = G_MAXINT;
   GDir *dp;
   const char *dentry;
   GError *error = NULL;
 
-  helpdir = dia_get_data_directory("help");
+  /* helpdir = dia_get_data_directory("help"); */
+  helpdir = g_strdup("/usr/share/doc/dia/html");
   if (!helpdir) {
     message_warning(_("Could not find help directory"));
     return;
@@ -696,7 +694,6 @@ help_manual_callback (GtkAction *action)
 #endif
 
   g_free(helpindex);
-#endif
 }
 
 static void 
--- dia-0.97.1.orig/xmldocs.make
+++ dia-0.97.1/xmldocs.make
@@ -36,13 +36,8 @@
 # **********  Begin of section some packagers may need to modify  **********
 # This variable (helpdocdir) specifies where the documents should be installed.
 # This default value should work for most packages.
-if HAVE_GNOME
-helpdocdir = $(datadir)/gnome/help/$(docname)/$(lang)
-install-data-hook: install-data-hook-omf
-else
-helpdocdir = $(datadir)/$(docname)/help/$(lang)
+helpdocdir = $(datadir)/doc/dia/html/$(lang)
 install-data-hook:
-endif
 
 # **********  You should not have to edit below this line  **********
 xml_files = $(entities) $(docname).xml
--- dia-0.97.1.orig/doc/Makefile.am
+++ dia-0.97.1/doc/Makefile.am
@@ -8,11 +8,7 @@ EXTRA_DIST = \
 	shape.dtd \
 	sheet.dtd
 
-if HAVE_GNOME
-helpdir = $(datadir)/gnome/help/$(docname)
-else
-helpdir = $(pkgdatadir)/help
-endif
+helpdir = $(datadir)/doc/dia/html
 
 sysdoc = $(DESTDIR)$(docdir)
 
